client/transactions/list/index.tsx

```diff
@@ -73,6 +77,26 @@ interface Column extends TableCardColumn {
 	cellClassName?: string;
 }
 
+const getPaymentSourceDetails = ( txn: Transaction ) => {
+	switch ( txn.source ) {
+		case 'giropay':
+			return <Fragment>{ txn.source_identifier }</Fragment>;
+		case 'p24':
+			return (
+				<Fragment>
+					{ p24BankList[ txn.source_identifier ] ?? '' }
+				</Fragment>
+			);
+		default:
+			return (
+				<Fragment>
+					&nbsp;&bull;&bull;&bull;&bull;&nbsp;{ ' ' }
```

Comment:
Can we replace these html entities with CSS? ;)

Reply:
I saw we use HTML in other places, so I think it's better to stick to that if there is no big difference in performance...